public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx

```diff
@@ -560,67 +602,7 @@ export const MetricEncyclopediaModal = (props: Props) => {
         </div>
       </div>
 
-      <div className={styles.results}>
-        {metrics &&
-          displayedMetrics(metrics).map((metric: MetricData, idx) => {
-            return (
-              <Collapse
-                aria-label={`open and close ${metric.value} query starter card`}
-                data-testid={testIds.metricCard}
-                key={metric.value}
-                label={metric.value}
-                isOpen={openTabs.includes(metric.value)}
-                collapsible={true}
-                onToggle={() =>
-                  setOpenTabs((tabs) =>
-                    // close tab if it's already open, otherwise open it
-                    tabs.includes(metric.value) ? tabs.filter((t) => t !== metric.value) : [...tabs, metric.value]
-                  )
-                }
-              >
-                <div className={styles.cardsContainer}>
```

Comment:
`cardsContainer`, `card` and `metadata` are all styles that are no longer used. these can be deleted from `getStyles()`

Reply:
Added!